setup.py

```diff
@@ -2,9 +2,12 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, print_function
 import os
+import re
 from setuptools import setup, find_packages
 
-from skipnose import __author__, __email__, __version__
+
+module_file = open("skipnose/__init__.py").read()
```

Comment:
can you please use `read()` from below which also makes sure `utf` decoding is done correctly.